node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -654,17 +746,26 @@ mod tests {
                 low: StdNumParamFormNamed::new(":balance_low_b".to_string(), 9223372036854321124)
             }
         );
-        assert_eq!(result.params_except_wei_change[0].value_pair.0, ":some_key");
         assert_eq!(
-            result.params_except_wei_change[0].value_pair.1.to_string(),
+            result.params_except_wei_change[0].param_pair().0,
```

Comment:
I think there are some repeated assertions, if we can eliminate one of them, it would be easier to understand the test, and I believe the assertions will still be strong enough to break the test in case something in the production changes.